src/button/button.test.js

```diff
@@ -0,0 +1,38 @@
+/* eslint-env jest */
+const puppeteer = require('puppeteer')
+
+let browser
+let page
+let baseUrl = 'http://localhost:3000'
+
+beforeAll(async () => {
+  browser = await puppeteer.launch()
+  page = await browser.newPage()
+})
+
+describe('/components/button', () => {
+  describe('/components/button/link', () => {
+    it('renders a link button', async () => {
+      await page.goto(baseUrl + '/components/button/link/preview', { waitUntil: 'load' })
+      const button = await page.evaluate(() => document.body.getElementsByTagName('a'))
+      expect(button).toBeTruthy()
+    })
+
+    it('triggers the click event when the space key is pressed', async () => {
+      await page.goto(baseUrl + '/components/button/link/preview', { waitUntil: 'load' })
+
+      const href = await page.evaluate(() => document.body.getElementsByTagName('a')[0].getAttribute('href'))
+
+      await page.keyboard.press('Tab')
```

Comment:
Would it be clearer / more explicit to use `page.focus('a')` here?

Reply:
Sure. I was replicating my manual testing, but it makes sense to use focus for this.